pkg/eraser/eraser.go

```diff
@@ -189,15 +198,41 @@ func removeVulnerableImages(c Client, socketPath string, imagelistName string) (
 		}
 	}
 
-	// TODO: change this to read vulnerable images from ImageList
-	// adding random image for testing purposes
+	// get vulnerable images from ImageList
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	result := eraserv1alpha1.ImageList{}
```

Comment:
i think we can revert these changes, as other pr covers them

Reply:
do current unit tests touch these?

Reply:
No, I only need the inferface above. I reverted these changes.